ssts/segmentation/example.py

```diff
@@ -89,6 +89,9 @@ def main(data_path=None, example_number=None):
         pca_components = seg.pca.transform(data.reshape(n, c))  # NOTE PCA was trained while fitting pre_seg
         pca_components = pca_components.reshape(h, w, num_pca_components)  # shape (512, 512, num_pca_components)
 
+        # Plot classification distributions of PCA components
+        mm.show_classification_distributions(pre_labels, pca_components, title_flag=False)
```

Comment:
Show classification distributions of PCA components. You can use this to see what components and properties dominate the classification